vent/api/actions.py

```diff
@@ -683,26 +683,42 @@ def get_configure(self,
                       groups=None,
                       enabled="yes",
                       branch="master",
-                      version="HEAD"):
+                      version="HEAD",
+                      main_cfg=False):
         """
         Get the vent.template settings for a given tool by looking at the
         plugin_manifest
         """
         self.logger.info("Starting: get_configure")
         constraints = locals()
         status = (True, None)
-        # all possible vent.template sections
-        options = ['info', 'service', 'settings', 'docker', 'gpu']
-        tools = self.p_helper.constraint_options(constraints, options)[0]
-        if tools:
-            # should only be one tool
-            tool = tools.keys()[0]
-            # load all vent.template options into dict
-            template_dict = {}
-            for section in tools[tool]:
-                template_dict[section] = json.loads(tools[tool][section])
-            # display all those options as they would in vent.template
-            return_str = ""
+        template_dict = {}
+        return_str = ""
+        if main_cfg:
+            vent_cfg_path = os.path.join(os.path.expanduser('~'), '.vent',
+                                         'vent.cfg')
+            if os.path.exists(vent_cfg_path):
+                vent_cfg = Template(vent_cfg_path)
+                for section in vent_cfg.sections()[1]:
+                    template_dict[section] = {}
+                    for vals in vent_cfg.section(section)[1]:
+                        template_dict[section][vals[0]] = vals[1]
+            else:
+                status = (False, "Couldn't get vent.cfg information")
+        else:
+            # all possible vent.template sections
+            options = ['info', 'service', 'settings', 'docker', 'gpu']
```

Comment:
By sections, I meant sections in vent.template, but for grabbing configuration information, I'm looking at the plugin_manifest, which stores those as options. I could loop through all the options for each section in plugin_manifest, but I'd have to eventually ask if those options are equal to one of the ones listed in the list, so I just used constrained_sections with those options. Sorry, I realize the wording is a little misleading and am changing that.